src/coordination/azure/Akka.Coordination.Azure/AzureLease.cs

```diff
@@ -92,12 +97,18 @@ public override async Task<bool> Release()
                 if(_log.IsDebugEnabled)
                     _log.Debug("Releasing lease");
                 var result = await _leaseActor.Ask(LeaseActor.Release.Instance, _timeout);
-                return result switch
+                switch (result)
                 {
-                    LeaseActor.LeaseReleased _ => true,
-                    LeaseActor.InvalidRequest req => throw new LeaseException(req.Reason),
-                    _ => throw new LeaseException($"Unexpected response type: {result.GetType()}")
-                };
+                        case LeaseActor.LeaseReleased:
+                            return true;
+                        case LeaseActor.InvalidReleaseRequest:
+                            _log.Info("Tried to release a lease that is not acquired");
+                            return true;
```

Comment:
Logically, since we didn't have the lease, we should return true.
This will cause the plugins to report that the lease have been successfully released, which can be confusing.
This is why the log should be emitted in info level so that we can track that the lease was "successfully released" because we didn't have it in the first place.